apps/app/src/pages/Build/ReviewButton.js

```diff
@@ -99,10 +113,32 @@ export function ReviewButtonContent({ repository }) {
   );
 }
 
+function DisabledReviewButton() {
+  const tooltip = useTooltipState();
+
+  return (
+    <>
+      <TooltipAnchor state={tooltip}>
+        <ReviewChangesButton disabled />
```

Comment:
So this usage is invalid, since you don't pass `state`.

Reply:
You are struggling because the whole Button + Menu should be in the same component, and it is that one that you disable.

Reply:
no more relevant.